### PR TITLE
Add Error Popup

### PR DESCRIPTION
**Pull request by Dauntless-Jacobson**

### CODE REVIEW

**Comment by Dauntless-Jacobson** [comment]
errorPropoargs

File: src/web-components/ErrorPopup/index.ts [hunk 1]
```diff
@@ -0,0 +1,246 @@
+import { LitElement, html, css } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+
```
Comment (bigirishlion) [inline src/web-components/ErrorPopup/index.ts:3]: extra line

File: src/web-components/ErrorPopup/index.ts [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+        .content-wrapper {
+            display: flex;
+            flex-direction: row;
+            justify-content: center;
+        }
+
+        .alert-icon {
+            display: flex;
+        }
+    `;
```
Comment (bigirishlion) [inline src/web-components/ErrorPopup/index.ts:149]: Can we move this to a separate css file?

**Review by bigirishlion — CHANGES_REQUESTED** [review]
🗡️ This is looking great! Can we do a few things?
- Add a "Title" option to match handle.me's error popup (See image)
- Add slot for custom HTML below message (See image)
- Move the close icon closer to the top right. (See image but don't match the black X. it should be white)
- Add matching margin on the top and bottom

image

**Comment by Dauntless-Jacobson** [comment]
errorpopup12

**Review by bigirishlion — APPROVED** [review]
🎲

**Pull request merged**